src/http/api/views/reply.chain.view.ts

```diff
@@ -66,6 +67,42 @@ export class ReplyChainView {
     constructor(private readonly db: Knex) {}
 
     private mapToPostDTO(result: PostRow, contextAccountId: number): PostDTO {
+        // If the post is deleted, return it as a tombstone
+        if (result.post_deleted_at !== null) {
+            return {
+                id: result.post_ap_id,
+                type: 2, // PostType.Tombstone
+                title: '',
+                excerpt: '',
+                summary: null,
+                content: '',
+                url: result.post_url,
+                featureImageUrl: null,
+                publishedAt: result.post_published_at,
+                likeCount: result.post_like_count,
+                likedByMe: result.post_liked_by_current_user === 1,
```

Comment:
I think we can leave this `false`. It doesn't make sense for the UI to show if a deleted post is liked/reposted. Same for the `repostedByMe` field below.

Reply:
Yeah, I think you're right, thanks!